app/controllers/assignments_controller.rb

```diff
@@ -31,6 +31,9 @@ def update_caseworker
     @project.update(caseworker_assigned_at: DateTime.now) if @project.caseworker_assigned_at.nil?
     @project.update(caseworker_params)
 
+    caseworker = User.find(caseworker_params["caseworker_id"])
+    CaseworkerAssignmentMailer.new_assigned_caseworker(caseworker, @project).deliver_later
```

Comment:
I think we can save the extra request here and do `CaseworkerAssignmentMailer.new_assigned_caseworker(@project.caseworker, @project).deliver_later`